sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -213,6 +213,7 @@ func runCtr(containerdSocket string, namespace string, containerID string, sourc
 			oci.WithHostNamespace(runtimespec.NetworkNamespace),
 			oci.WithHostHostsFile,
 			oci.WithHostResolvconf,
+			oci.WithEnv([]string{"HTTPS_PROXY=" + os.Getenv("HTTPS_PROXY"), "NO_PROXY=" + os.Getenv("NO_PROXY")}),
```

Comment:
Should we check whether `HTTPS_PROXY` is set before adding it to the environment? Applications might handle "set but empty" and "unset" differently.

Should we also export `https_proxy` and `no_proxy`, as we've done in `proxy.env`?